.123trigger

```diff
@@ -1,3 +1,3 @@
 0.1.4
 0.1.50.1.80.1.80.1.90.1.90.1.90.1.100.1.110.1.110.1.110.1.12
-0.1.120.1.130.1.130.1.130.1.130.1.140.1.140.1.150.1.150.1.160.1.160.1.170.1.170.1.170.1.170.1.190.1.190.2.20.2.3
\ No newline at end of file
+0.1.120.1.130.1.130.1.130.1.130.1.140.1.140.1.150.1.150.1.160.1.160.1.170.1.170.1.170.1.170.1.190.1.190.2.20.2.30.2.4
```

Comment:
Why do we have this file in the project?

Reply:
It's according to what Nikhil advised. The file is used in https://github.com/AstarNetwork/astar.js/blob/main/scripts/build-release.js#L124